x-pack/plugins/cloud_security_posture/public/pages/findings/utils.ts

```diff
@@ -8,14 +8,17 @@
 import { buildEsQuery } from '@kbn/es-query';
 import type { DataView } from '@kbn/data-plugin/common';
 import { EuiBasicTableProps, Pagination } from '@elastic/eui';
-import { useEffect, useMemo } from 'react';
+import { useCallback, useEffect, useMemo } from 'react';
 import { i18n } from '@kbn/i18n';
 import type { FindingsBaseURLQuery } from './types';
 import { useKibana } from '../../common/hooks/use_kibana';
 
-export const getDefaultUrlQuery = (): FindingsBaseURLQuery => ({
-  query: { language: 'kuery', query: '' },
-  filters: [],
+export const getDefaultUrlQuery = ({
+  query = { language: 'kuery', query: '' },
+  filters = [],
```

Comment:
technically, all consumers pass a `query` and `filters` because all of them explicitly persist app filters. 

but one could only partial persist if needed be